content/en/tracing/connect_rum_and_traces/_index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Connect RUM and Traces
+kind: documentation
+further_reading:
+  - link: "https://www.datadoghq.com/blog/real-user-monitoring-with-datadog/"
+    tag: "Blog"
+    text: "Real User Monitoring"
+  - link: "/tracing/"
+    tag: "Documentation"
+    text: "APM and Distributed Tracing"
+---
+
+{{< img src="tracing/connect_rum_and_traces/rum_trace_tab.png" alt="RUM and Traces"  style="width:100%;">}}
+
+# Overview
+
+The APM integration with Real User Monitoring allows you to link requests from your web and mobile applications to their corresponding backend traces, while providing the RUM session context.
+
+Having frontend-related specifics (thanks to your RUM session) as well as backend, infrastructure, and log information (thanks to your trace id injection) allows you to pinpoint issues anywhere in your stack in unprecedented speed with a full understanding of what your users are experiencing.
+
+<div class="alert alert-warning">
+This feature is currently available for private beta. <a href="https://docs.datadoghq.com/help/">Reach out to support</a> to turn on this feature for your account.
+</div>
+
+# Usage
+## Prerequisites
+
+-   Your services targeted by your RUM applications are [traced on the APM side][1].
+-   Your services use an HTTP server.
+-   Your HTTP servers are using a library that supports distributed tracing.
```

Comment:
do we have a list of these that we can link to? I am not really sure what you mean here

Reply:
Should we link to the section underneath?

Reply:
yes, perfect